third_party/move/tools/move-formatter/tests/formatter/spec_struct/input5.move.fmt

```diff
@@ -0,0 +1,20 @@
+/// test_point: Struct ability written on multiple lines
+
+/// The `ASCII` module defines basic string and char newtypes in Move that verify
+/// that characters are valid ASCII, and that strings consist of only valid ASCII characters.
+module std::ascii {
+    use std::vector;
+    use std::option::{Self, Option};
+    
+    /// An ASCII character.
+    struct Char has copy, /* comment */ /* comment */ drop,  // comment
```

Comment:
Comments that were standalone in their own line have been pulled up and added into the same line, this can have unintended consequences and is unexpected as far as the user is concerned, especially the last `// comment`.

Reply:
Yes, this is an error, we have updated it